app/models/organization.rb

```diff
@@ -44,7 +44,7 @@ def alternate_names
   end
 
   def names
-    organization_names.order(created_at: :desc).sort_by(&:rank).pluck(:content)
+    organization_names.sort_by { |n| [-n.created_at.to_i, n.rank] }.pluck(:content)
```

Comment:
this query also exists [here](https://github.com/cavvia/bearden/blob/master/app/models/organization_resolver.rb#L29), so we would need to update that too.

Reply:
I consolidated ranking into `OrganizationResolver`.